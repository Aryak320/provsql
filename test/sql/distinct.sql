\set ECHO none
\pset format unaligned
SET search_path TO provsql_test, provsql;

CREATE TABLE distinct_result AS
  SELECT *, sr_formula(provenance(),'personnel_name') AS formula
  FROM (
    SELECT DISTINCT classification FROM personnel
  ) t;

SELECT remove_provenance('distinct_result');
SELECT classification,replace(formula,'(Paul ⊕ Nancy)','(Nancy ⊕ Paul)') AS formula FROM distinct_result ORDER BY classification;
DROP TABLE distinct_result;
