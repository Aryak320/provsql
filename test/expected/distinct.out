\set ECHO none
remove_provenance

classification,formula
unclassified,John
restricted,(Nancy ⊕ Paul)
confidential,Dave
secret,(Ellen ⊕ Susan)
top_secret,Magdalen
